### ISSUE TITLE
question

### ISSUE THREAD

**Issue by dong-0412** [mshukor/TFood#4]
`dataset = factory(split)
    if Options()['misc'].get("device_id", False):
        device_id =  Options()['misc.device_id']
    else:
        device_id = 0
    Logger()('Load model...')
    model = model_factory()
    model_state = torch.load(path_model_ckpt)
    model.load_state_dict(model_state)
    model.eval()`

the code above from the ingrs_to_images.py ,I confused with it , that the dataset and model loadeling process maybe roung , could you give me some guide about it, or about the retrival process

**Comment by mshukor** [comment]
Some files (including this `ingrs_to_images.py`) are not used in TFood, they are from the original Adamine repo. The functions that compute the metrics for retrieval (and hence used during test) is in `models.metrics.trijoint..py`. You can follow the instructions in the README to do the test.

**Comment by dong-0412** [comment]
Thank you for your reply , I am a beginner in NL , I run the test process , jiust like the "Evaluate a model on the test set" in README  , but it jiut only caculate some embeddings and features when the program finished , I am still confused about how to process the retrival , could you please give me some guidence

**Comment by dong-0412** [comment]
Thank you for your reply , I am a beginner in NL , I run the test process , jiust like the "Evaluate a model on the test set" in README , but it just only caculate some embeddings and features when the program finished , I am still confused about how to process the retrival , could you please give me some guidence

**Comment by dong-0412** [comment]
thank you for reply , I'm a beginner in NL , and I'd like to know have you finished the retieval process for TFood , just like the ingrs_to_images.py in Adamine repo which can retrieval some images by a ingredient input array.
